lib/testLauncher/SuitestLauncher.js

```diff
@@ -36,6 +37,62 @@ class SuitestLauncher {
 		});
 	}
 
+	async runTokenSession() {
+		suitest.logger.intro(launcherGreeting, version, TOKEN);
+		validateInput(TOKEN.toUpperCase(), this.ownArgv);
+
+		try {
+			/* istanbul ignore if  */
+			if (isDebugMode(this.ownArgv)) {
+				// debugging not allowed in automated session
+				throwDebugInAutomatedError();
```

Comment:
@JeKachan Debugging is actually allowed. Please refer ticket description and discussion in https://mineus.atlassian.net/browse/SUIT-17313. Because of merging automated and interactive mode debugging is possible with all kinds of tokens.